app/Platform/RouteServiceProvider.php

```diff
@@ -71,6 +72,7 @@ protected function mapWebRoutes(): void
             // Route::get('game/{game}/players', [GamePlayerController::class, 'index'])->name('game.player.index');
             Route::get('game/{game}/dev-interest', GameDevInterestController::class)->name('game.dev-interest');
             Route::get('game/{game}/suggest', [SuggestGameController::class, 'forGame'])->name('game.suggest');
+            Route::get('game/{game}/achievements/compare/{user}', CompareUnlocksController::class)->name('game.compare-unlocks');
```

Comment:
I think I'd go with something like `/user/{user}/game/{game}/compare/{targetUser}` as it's about two users where the game is related to the first as a sub-resource.

Reply:
The first user is implied as the logged in user. Do we want to allow arbitrarily comparing any two users?

`/user/me/game/1/compare/them`
`/user/them/game/1/compare/someoneelse`

Reply:
Good question. I guess it's enough to only ever compare the currently logged-in user with the user specified via URL.

So it's just the inversion of the current route: `/user/{user}/game/{game}/compare`
If I'm not mistaken it would allow to keep the current implementation of `CompareUlocksController` as-is?

Reply:
There's a few things that had to be altered (particularly the path to the file and the breadcrumb), but I've made the suggested changes.